### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
ping @jmchilton

File: galaxy.xsd [hunk 1]
```diff
@@ -295,12 +294,12 @@ resolver.
...
         <xs:attribute name="type" type="RequirementType" use="required">
           <xs:annotation>
-            <xs:documentation xml:lang="en"> This value defines the which type of the 3rd party module required by this tool </xs:documentation>
+            <xs:documentation xml:lang="en"> This value defines the which type of the 3rd party module required by this tool. </xs:documentation>
           </xs:annotation>
         </xs:attribute>
         <xs:attribute name="version" type="xs:string">
           <xs:annotation>
-            <xs:documentation xml:lang="en"> Required for package type requirements </xs:documentation>
+            <xs:documentation xml:lang="en"> For package type requirements this value defines a specific version of the tool depdendency. </xs:documentation>
```
Comment (nsoranzo) [inline galaxy.xsd]: s/depdendency/dependency/

File: galaxy.xsd [hunk 2]
```diff
@@ -519,7 +518,7 @@ def get_field_components_options( dataset, field_name ):
     <xs:annotation gxdocs:best_practices="tests">
       <xs:documentation xml:lang="en"><![CDATA[
 
-Container tag set for the <test> tag sets. Any number of tests can be included,
+Tag set to specify tests via the <test> tag sets. Any number of tests can be included,
```
Comment (nsoranzo) [inline galaxy.xsd]: s/Tag set/Container tag/ maybe?

File: galaxy.xsd [hunk 3]
```diff
@@ -1340,7 +1339,7 @@ conditionally reference ``${input_type.process_obs_metadata}`` with a Cheetah
 ``if`` statement.
 
 A common use of the conditional wrapper is to select between reference data
-(managed by the Galaxy deployer or admin - for instance via Data Managers) or
+managed by the Galaxy deployer or admin - for instance via Data Managers or
```
Comment (nsoranzo) [inline galaxy.xsd:1342]: `managed by the Galaxy admins (for instance via Data Managers) and`

File: galaxy.xsd [hunk 4]
```diff
@@ -2099,13 +2115,12 @@ code file wraps a metadata collection to return metadata parameters wrapped
 according to the Metadata spec.
 
 ```xml
-<command>
```
Comment (nsoranzo) [inline galaxy.xsd:2102]: I'd restore `<command>` here and also add `<![CDATA[` (and correspondingly at the end).

**Review by nsoranzo — CHANGES_REQUESTED** [review]

File: galaxy.xsd (hunk 4 above)
Comment (bgruening) [reply galaxy.xsd:2102]: You can not use a CDATA in a CDATA isn't it?

Comment (nsoranzo) [reply galaxy.xsd:2102]: I didn't notice that was already inside a CDATA, but it's possible: https://en.wikipedia.org/wiki/CDATA#Nesting

Comment (jmchilton) [reply galaxy.xsd:2102]: I tried the nested CDATA thing somewhere else and it didn't work for me. I can try again though.

**Comment by jmchilton** [comment]
Sorry I didn't merge this before opening the Galaxy PR :(. I'm going to work on a few more things and just copy the galaxy.xsd over to that PR - so you'll loose authorship of the commit. If you want me to rebuild and include your commits with authorship let me know.

**Comment by jmchilton** [comment]
Also thanks for the changes! And thanks for the detailed review as always @nsoranzo!

**Comment by bgruening** [comment]
I don't care. Go!

**Pull request merged**